app/components/Search/ResourcesTable.js

```diff
@@ -138,19 +124,14 @@ class ResourcesTable extends Component {
   }
 
   componentDidMount() {
-    if(this.props.userLocation) {
-      let position = this.props.userLocation;
-      let userLocation = {lat: position.coords.latitude, lng: position.coords.longitude};
-      this.setState({
-        location: userLocation
-      });
-      this.loadResourcesFromServer(userLocation);
-    } else {
-      let self = this;
-      this.getLocationGoogle()
-        .then(loc => {
-          self.loadResourcesFromServer(loc);
-        });
+    if (this.props.userLocation) {
```

Comment:
Probably fine to take this out of the if - there's a hard-coded location passed down and I think the api can handle requests without coords, anyway.